great-idea/index.html

```diff
@@ -16,8 +16,79 @@
 
   <body>
 
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+    <nav>
+  <a href="#">Services</a>
```

Comment:
Don't forget to link `Service` and the logo to link to `service.html` and `index.html` respectively!